server/hsts.go

```diff
@@ -0,0 +1,11 @@
+package server
+
+import "net/http"
+
+// HTTP Strict Transport Security
+func HSTS(h http.Handler) http.Handler {
+	return http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
+		rw.Header().Add("Strict-Transport-Security", "max-age=31536000; includeSubDomains")
```

Comment:
Please make these configurable:

1. `max-age` value
2. `includeSubDomains` enabled/disabled

And then you can probably use `max-age > 0` instead of `EnableHSTS`.